### DISCUSSION TITLE
Welcome to Canadian-Tech-Internships-Summer-2024 Discussions!

### DISCUSSION THREAD

**Discussion by jenndryden** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#74] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nilesh1168** [comment]
Hello ! My name is Nilesh. I am currently pursuing Master of Applied Computer Science degree at Concordia University, Montreal, Quebec, Canada. I am international student and in 2nd semester of my program. Currently, I am passively searching for internships in the software industry and appreciate the help I get from this repository.

**Reply by jenndryden** [reply]
Hi Nilesh! Nice to meet you! :)

**Reply by kanjurer** [reply]
Hi Nilesh. Happy job hunting
